spec/support/specialist_publisher_assertion_helpers.rb

```diff
@@ -29,5 +35,17 @@ def expect_unpublished
     expect(find(".alert").text).to match(/^Unpublished/)
   end
 
+  def reload_while_failing
+    attempts = 1
+    begin
+      yield
+    rescue Capybara::ElementNotFound, RSpec::Expectations::ExpectationNotMetError => e
+      raise e if attempts >= Capybara.default_max_wait_time
```

Comment:
I'd prefer using a simple value (e.g. 3 say), as `Capybara.default_max_wait_time` is a duration in seconds if I'm not mistaken, which has a different meaning.